infrastructure/kubernetes/app-backend.yml

```diff
@@ -58,7 +58,14 @@ spec:
               name: app-backend-secrets
               key: pre_shared_key
         - name: "REDIS_URL"
-          value: "redis://localhost"
+          value: "io-development.redis.cache.windows.net"
```

Comment:
We must add the redis resource through Terraform and put here the "definitive" address, may you please open a story for this ?

Reply:
see https://github.com/teamdigitale/digital-citizenship/pull/138